include/aws/crt/mqtt/Mqtt5Client.h

```diff
@@ -595,15 +595,27 @@ namespace Aws
                  */
                 Mqtt5ClientOptions &WithConnackTimeoutMs(uint32_t connackTimeoutMs) noexcept;
 
+                /**
+                 * @deprecated The function is deprecated, please use `Mqtt5ClientOptions::WithAckTimeoutSec(uint32_t)`
```

Comment:
I'm confused.  Why are we adding a function and deprecating it at the same time?

Reply:
Ah nm, diff was confusing